.changeset/warm-hotels-happen.md

```diff
@@ -0,0 +1,5 @@
+---
+'storybook': minor
+---
+
+Created a new example of how to use the Dialog component in Storybook
```

Comment:
As it's just a Storybook story, you don't need to add a changeset. This change won't be noticeable to package consumers.

Reply:
ok, thanks! :)